src/ProcessManagerBundle/Resources/config/services.yml

```diff
@@ -95,3 +104,11 @@ services:
             - '@process_manager.registry.processes'
         tags:
             - { name: messenger.message_handler }
+
+    ### Cleanup service
+    ProcessManagerBundle\Service\CleanupService:
+        arguments:
+            - '@parameter_bag'
+
+    # service IDs used by the codebase directly
+    process_manager.cleanup_service: '@ProcessManagerBundle\Service\CleanupService'
```

Comment:
not needed

Reply:
Currently used to get the service without DI in `ProcessController`.